code/modules/mob/living/silicon/robot/robot.dm

```diff
@@ -1,5 +1,5 @@
 #define CYBORG_POWER_USAGE_MULTIPLIER 2.5 // Multiplier for amount of power cyborgs use.
-
+var/malfAImoduleavailable = 0

```

Comment:
Hrm. Can we somehow do without the global variable? Perhaps by accessing the gamemode? If not, then, RIP.